dashboard-for-web/articles/web-dashboard-designer-mode/designing-dashboard-items/chart.md

```diff
@@ -29,4 +29,7 @@ This section is divided into the following subsections.
 	Describes how to customize settings related to chart axes.
 * [Orientation](chart/orientation.md)
 	
```

Comment:
Please change the following lines above as follows:

Line 7:
`The topics in this section describe the features available in the **Chart** dashboard item, and explain how to create and customize charts in the Web Dashboard.`

Line 14:
`Describes how to supply the Chart dashboard item with data.`

Line 26:
`Describes the chart legend and its options.`